test/runtests.jl

```diff
@@ -1,8 +1,10 @@
-using AstroImages, FITSIO, Images
+using AstroImages, FITSIO, Images, Random
 using Test
 
 import AstroImages: _float, render
 
+fname = tempname() * ".fits"
```

Comment:
I'm confused, now this file is not removed at the end any more?

Reply:
I was suspicious of `rm` causing the `EBUSY` error on appvyeor. So I thought using it just once should be good enough. Fixed in the latest commit, seems to be working well that way too.

Reply:
Well, there was no `rm` at all in https://github.com/JuliaAstro/AstroImages.jl/blob/024a521f09cf590a34b7335605300b394a539346/test/runtests.jl :wink: